stetho/src/main/java/com/facebook/stetho/inspector/elements/android/ViewDescriptor.java

```diff
@@ -10,19 +10,147 @@
+      synchronized (this) {
+        if (VIEW_GETTERS == null) {
+          Set<String> allowedGetters = new HashSet<>(
+              Arrays.asList(
+                  "getHorizontalScrollFactor",
+                  "getBottom",
+                  "getTopFadingEdgeStrength",
+                  "getPaddingBottom",
+                  "getImportantForAccessibility",
+                  "getScrollX",
+                  "getScrollY",
+                  "getWindowVisibility",
+                  "getId",
+                  "getRawTextAlignment",
+                  "getWindowSystemUiVisibility",
+                  "getRotation",
+                  "getVerticalScrollbarPosition",
+                  "getRightPaddingOffset",
+                  "getBottomPaddingOffset",
+                  "getMeasuredHeight",
+                  "getBottomFadingEdgeStrength",
+                  "getPaddingEnd",
+                  "getRawTextDirection",
+                  "getTextDirection",
+                  "getVisibility",
+                  "getHorizontalFadingEdgeLength",
+                  "getLeftPaddingOffset",
+                  "getScaleY",
+                  "getScaleX",
+                  "getScrollBarFadeDuration",
+                  "getLayoutDirection",
+                  "getLeft",
+                  "getMeasuredWidthAndState",
+                  "getVerticalScrollbarWidth",
+                  "getPaddingRight",
+                  "getPaddingStart",
+                  "getLeftFadingEdgeStrength",
+                  "getSolidColor",
+                  "getScrollBarDefaultDelayBeforeFade",
+                  "getAlpha",
+                  "getVerticalFadingEdgeLength",
+                  "getHorizontalScrollbarHeight",
+                  "getRightFadingEdgeStrength",
+                  "getWidth",
+                  "getTop",
+                  "getSuggestedMinimumHeight",
+                  "getSuggestedMinimumWidth",
+                  "getKeepScreenOn",
+                  "getPaddingLeft",
+                  "getMeasuredHeightAndState",
+                  "getTranslationX",
+                  "getTranslationY",
+                  "getPaddingTop",
+                  "getPivotY",
+                  "getPivotX",
+                  "getRotationY",
+                  "getRawLayoutDirection",
+                  "getRotationX",
+                  "getScrollBarStyle",
+                  "getMinimumHeight",
+                  "getMeasuredState",
+                  "getHeight",
+                  "getTextAlignment",
+                  "getTopPaddingOffset",
+                  "getMinimumWidth",
+                  "getVerticalScrollFactor",
+                  "getScrollBarSize",
+                  "getMeasuredWidth",
+                  "getWindowAttachCount",
+                  "getBaseline",
+                  "getY",
+                  "getX",
+                  "getRight"
+              ));
+
+          Method[] viewMethods = View.class.getDeclaredMethods();
```

Comment:
Something I just realized we're not handling here: We only look at `View.class.getDeclaredMethods`. We are not picking up anything from derived classes, e.g. TextView. Either this doesn't matter, or we need to find a way to do it (or just decide not to :) )